src/components/Notifications/TimeConfig.tsx

```diff
@@ -52,27 +60,25 @@ export const TimeConfigComponent: React.FunctionComponent = () => {
         setShowCustomSelect(true);
     }, []);
 
-    const handleTimePrefSelect = React.useCallback(() => {
-        setTimeSelect(timeSelect);
+    const handleTimePrefSelect = React.useCallback((time) => {
+        setTimeSelect(time);
         setIsOpen(false);
-    }, [ timeSelect ]);
+    }, [ ]);
 
     const handleButtonSave = React.useCallback(() => {
-        const mutate = saveTimePreference.mutate;
-        mutate({
-            body
-        }).then((response) => {
-            if (response.status === 200) {
-                return (
-                    <Alert title='Action settings saved' variant='success' />
-                );
-            } else {
-                return (
-                    <Alert title='Failed to save action settings' variant='danger' />
-                );
-            }
-        });
-    }, [ saveTimePreference.mutate ]);
+        if (timeSelect) {
+            const mutate = saveTimePreference.mutate;
+            mutate({
+                body: timeSelect
+            }).then((response) => {
+                if (response.status === 200) {
+                    addSuccessNotification('Action settings saved', '');
```

Comment:
Updated the way to add toast messages. I'm assuming we are doing this differently than other apps?